extensions/git/src/commands.ts

```diff
@@ -17,15 +17,21 @@ import * as nls from 'vscode-nls';
 
 const localize = nls.loadMessageBundle();
 
-class CheckoutItem implements QuickPickItem {
+class CheckoutBasicItem implements QuickPickItem {
```

Comment:
Why not instead of this simply create a new `class MergeItem implements QuickPick`?